templates/cloud_sidebar.hbs

```diff
@@ -103,8 +105,19 @@
                 </ul>
             </div>
 
-            <div class="accordion-menu-item">
+             <div class="accordion-menu-item">
                 <input type="checkbox" id="rd6" name="rd">
+                <label for="rd5" class="accordion-menu-item-label menu-label">
```

Comment:
Trying this in preview, the Private Beta Features accordion never opens.  I think this is down to the label on this line and the id on the previous line not matching.  And something is messed up on lines 120-121 where the label there matches the ID here... so clicking Contributing opens Private Beta Features.  And clicking Private Beta Features opens Support.  Whee!

Also, the one link inside this folder is broken.